### DISCUSSION TITLE
Native super table field

### DISCUSSION THREAD

**Discussion by angrybrad** [craftcms/cms#9384] [Ideas]
> Created by: Steven Grant (hello@digitalevangelist.net) on 2015/03/10 17:09:13 +0000
> Votes at time of UserVoice import: 4

-----------------------------------

Whilst Matrix is great, I think it can sometimes be a bit too much for a particular piece of content. I think there's definitely scope for a field that offered exactly what Matrix originally offered to ExpressionEngine, ie a grid that had configurable column types.

https://github.com/engram-design/SuperTable is exactly that but again, feel this should be something native.

**Comment by KatieMFritz** [comment]
I find myself using SuperTable static fields quite a bit to group related fields (for example, an address or settings sets). I guess it's like a matrix with one block/row?

**Comment by brandonkelly** [comment]
@KatieMFritz That sort of thing will be possible just using field layout features in 4.0 :) (#2644)

**Comment by brandonkelly** [comment]
Worth mentioning here that as of Craft 3.5, you will be able to place multiple fields side-by-side (see https://github.com/craftcms/cms/issues/2644#issuecomment-656955838), including within Matrix fields (see https://github.com/craftcms/cms/issues/6346#issuecomment-658919451). Normal field layouts will also be able to include headings, tips, instruction text, horizontal rules, and even custom UI elements based on site templates. All of that should help reduce the need to use Matrix/Super Table for presentational enhancements.
